packages/package-builder/src/Contract/HttpKernel/Contract/ExtraConfigAwareKernelInterface.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Symplify\PackageBuilder\Contract\HttpKernel;
+namespace Symplify\PackageBuilder\Contract\HttpKernel\Contract;
```

Comment:
@TomasVotruba I think the moving to **Contract** namespace should be skipped if the interface itself already have `\Contract\` namespace, so: the interface named _Symplify\PackageBuilder\Contract\HttpKernel\ExtraConfigAwareKernelInterface_ should be skipped, do you agree?

Reply:
see https://github.com/rectorphp/rector/pull/4554

Reply:
Agreed